### PR TITLE
Add rdp1070.yaml

### PR DESCRIPTION
**Pull request by rdp1070**

### CODE REVIEW

File: people/2016/spring/rdp1070.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Bobby D. Pruden
+blog: https://rdp1070.wordpress.com/
+feed: https://rdp1070.wordpress.com/feed/
+email: rdp1070@rit.edu
+major: New Media Interactive Development 2017
+irc: JustBobbyThings
+forges:
```
Comment (Nolski) [inline people/2016/spring/rdp1070.yaml]: This line should look like this
```yaml
forges:
  - https://github.com/rdp1070
```

**Comment by Nolski** [comment]
Almost there! If you make the change to your `forges` part of your yaml and then commit and push those changes to your current branch, you should be good to go.

File: people/2016/spring/rdp1070.yaml (hunk 1 above)
Comment (rdp1070) [inline people/2016/spring/rdp1070.yaml]: Does this work too? 
`forges:
  - GitHub: https://github.com/rdp1070
`

Comment (Nolski) [inline people/2016/spring/rdp1070.yaml]: Nope, I believe it has to be represented like a yaml list where each value is denoted by a `- keyvalue`

P.S. if you want to type code of a certain language you can use triple backtick at the beginning and end of your code that you have pasted.

**Comment by Nolski** [comment]
\o/

**Pull request merged**